src/user/user.controller.ts

```diff
@@ -4,19 +4,24 @@ import {
   Param,
   UseGuards,
   ParseUUIDPipe,
+  Patch,
+  Body,
 } from '@nestjs/common';
 import { UserService } from './user.service';
 import { AuthGuard } from 'src/auth/guards/auth.guard';
 import { User } from '@prisma/client';
 import { CurrentUser } from 'src/auth/decorators/current.user.decorator';
+import { UpdateUserDto } from './dto/update-user.dto';
 
 @UseGuards(AuthGuard)
 @Controller('')
```

Comment:
Consider adding a `users` prefix to this controller
```suggestion
@Controller('users')
```